Tribler/Test/Core/TFTP/test_tftp_handler.py

```diff
@@ -0,0 +1,160 @@
+from nose.tools import raises
+from Tribler.Core.TFTP.exception import FileNotFound
+from Tribler.Core.TFTP.handler import TftpHandler
+from Tribler.Core.TFTP.packet import OPCODE_OACK, OPCODE_ERROR
+from Tribler.Test.Core.base_test import TriblerCoreTest, MockObject
+from Tribler.dispersy.util import blocking_call_on_reactor_thread
+
+
+class TestTFTPHandler(TriblerCoreTest):
+    """
+    This class contains tests for the TFTP handler class.
+    """
+
+    def setUp(self, annotate=True):
+        TriblerCoreTest.setUp(self, annotate=annotate)
+        self.handler = TftpHandler(None, None, None)
+
+    @blocking_call_on_reactor_thread
```

Comment:
Yes, `the download_file` method that's called in the test is decorated by a `call_on_reactor_thread` and I want to wait until this method has been completely executed so that's why I decorated the test.